tuna/miopen/miopen_lib.py

```diff
@@ -163,7 +170,7 @@ def parse_args(self):
       print_solvers()
       raise ValueError('Printing solvers...')
 
-    if self.args.fin_steps:
+    if self.args.fin_steps and self.args.subcommand != 'load_job':
```

Comment:
```suggestion
    if self.args.fin_steps and self.args.subcommand and  self.args.subcommand != 'load_job':
```
you need to check if there is a subcommand first before u check the name otherwise u rist self.args.subcommand is None

Reply:
It would error out under that case